kit/azure/organization-hierarchy/main.tf

```diff
@@ -7,7 +7,7 @@ resource "azurerm_management_group" "parent" {
 module "policy_root" {
   source = "github.com/meshcloud/collie-hub//kit/azure/util/azure-policies?ref=da8dd49"
 
-  policy_path         = "./lib/root"
+  policy_path         = "./lib"
```

Comment:
d: btw I know there's [path.module](https://developer.hashicorp.com/terraform/language/expressions/references#filesystem-and-workspace-info) in terraform, not sure if we should be using this instead of `./`?